autoload/SpaceVim/layers/edit.vim

```diff
@@ -32,6 +32,7 @@ function! SpaceVim#layers#edit#plugins() abort
         \ [g:_spacevim_root_dir . 'bundle/vim-jplus', { 'on_map' : '<Plug>(jplus' }],
         \ [g:_spacevim_root_dir . 'bundle/tabular',           { 'merged' : 0}],
         \ [g:_spacevim_root_dir . 'bundle/vim-better-whitespace',  { 'on_cmd' : ['StripWhitespace', 'ToggleWhitespace', 'DisableWhitespace', 'EnableWhitespace']}],
+        \ ['andrewradev/splitjoin.vim',{ 'merged' : 0, 'loadconf' : 1}],
```

Comment:
Maybe I should add this plugin to "bundle" ?

Reply:
Yeah, because this layer is enabled by default, all plugins in this layer should be included in bundle dir.